src/utils.js

```diff
@@ -15,7 +15,9 @@ import differenceInCalendarMonths from 'date-fns/differenceInCalendarMonths';
 export function composeEventHandlers(...fns) {
   return (event, ...args) =>
     fns.some(fn => {
-      fn && fn(event, ...args);
+      if (fn) {
+        fn(event, ...args);
+      }
```

Comment:
FYI: this project has no lock file.
It looks like that some lint rules have changed.

```sh
✖ kcd-scripts lint found some errors. Please fix them and try committing again.

/Users/andy/work/js/dayzed/src/utils.js
  18:7  error  Expected an assignment or function call and instead saw an expression  babel/no-unused-expressions

✖ 1 problem (1 error, 0 warnings)
```